doxygen/mbedtls.doxyfile

```diff
@@ -1594,7 +1594,7 @@ SEARCH_INCLUDES        = YES
 # contain include files that are not input files but should be processed by
 # the preprocessor.
 
-INCLUDE_PATH           =
+INCLUDE_PATH           = ../include
```

Comment:
There is a typo in the commit message: efectively instead of effectively.

Reply:
Thanks for spotting it! Fixed now.